src/Http/Response.php

```diff
@@ -2233,6 +2233,21 @@ public function withFile($path, array $options = [])
     }
 
     /**
+     * Convenience method to set a string into the response body
+     *
+     * @param string $string The string to be sent
+     * @return static
+     */
+    public function withStringBody($string)
+    {
+        $new = clone $this;
+        $new->_createStream();
```

Comment:
> $this and $new used some class and this method used within this class

Yes I know that but I find it strange PHP allows accessing protected method of other objects just cause you are trying to access it from same class. Wonder how other languages handle such situation.